tox.ini

```diff
@@ -1,13 +1,12 @@
 [tox]
-envlist = py27, py35, py36, py37, py38, mypy
+envlist = py38, mypy, py39, py310
```

Comment:
The ordering is weird here. Is it on purpose?

Reply:
Yes: mypy runs on Python 3.8 (same as before), so I prefer that `mypy` is next to `py38`.